src/www/ui_components/server-list.html

```diff
@@ -49,6 +49,8 @@
         server-name="[[item.name]]"
         server-host="[[item.host]]"
         server-port="[[item.config.port]]"
+        server-cipher="[[item.config.method]]"
+        uses-unsupported-cipher="[[item.usesUnsupportedCipher]]"
```

Comment:
This will changes with the suggestions above

Reply:
Ack